src/command.ts

```diff
@@ -255,10 +260,27 @@ export default abstract class Command {
       if (deprecated) {
         this.warn(formatFlagDeprecationWarning(flag, deprecated))
       }
+
+      const deprecateAliases = this.ctor.flags[flag]?.deprecateAliases
+      const aliases = this.ctor.flags[flag]?.aliases ?? []
+      if (deprecateAliases && aliases) {
+        const foundAliases = this.argv.filter(a => aliases.includes(a.replace(/-/g, '')))
```

Comment:
what's happening with the `-` ?

Reply:
argv has the `--`/`-` on all the flag names e.g., `['--foo', '--bar', '-b']`

But the flag aliases don't have the prefix, e.g. `['foo', 'bar', 'b']`

So in order to find the intersection, we have to remove all the `-`s first

Reply:
what happens for `target-org` and `no-prompt` and the like?